javascript/javascript1/week2/StudentManager/script.js

```diff
@@ -0,0 +1,38 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+  // You write code here
+
+    if (studentName === "") {
+        return "You should add a name";
+    }
+
+    if (studentName === "Queen") {
+        class07Students.push("Queen");
+        return;
+      }
+
+    else if (studentName !== "Queen" && class07Students.length >= 6) {
+        return "Max slots are reached";
+    }
+    
+
+    else if (class07Students.length < 6 && !class07Students.includes(studentName) ) {
```

Comment:
Good handling of the different outcomes! The assignment states that you should log a message if the student is already present, so it sadly not compliant. 🥺 That should be an easy fix for you though! 😄 

With that being said, I think your design could also be sensible as getting the same output whether you are added or already present does make sense. However, it is important to follow the specification provided as someone calling your code might be surprised if it differs in behavior! 😱